packages/react-components/react-popover/src/components/PopoverTrigger/usePopoverTrigger.ts

```diff
@@ -70,16 +65,19 @@ export const usePopoverTrigger_unstable = (props: PopoverTriggerProps): PopoverT
   };
 
   return {
-    children: applyTriggerPropsToChildren<PopoverTriggerChildProps>(props.children, {
-      ...triggerAttributes,
-      'aria-expanded': `${open}`,
-      ...child?.props,
-      onClick: useEventCallback(mergeCallbacks(child?.props?.onClick, onClick)),
-      onMouseEnter: useEventCallback(mergeCallbacks(child?.props?.onMouseEnter, onMouseEnter)),
-      onKeyDown: useEventCallback(mergeCallbacks(child?.props?.onKeyDown, onKeyDown)),
-      onMouseLeave: useEventCallback(mergeCallbacks(child?.props?.onMouseLeave, onMouseLeave)),
-      onContextMenu: useEventCallback(mergeCallbacks(child?.props?.onContextMenu, onContextMenu)),
-      ref: useMergedRefs(triggerRef, child?.ref),
-    }),
+    children: applyTriggerPropsToChildren<PopoverTriggerChildProps>(
+      props.children,
```

Comment:
the same prop spread needs to happen here as for `MenuTrigger` since this component supports context menu